astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -29,6 +29,10 @@
 from .baseradec import BaseRADecFrame
 from .cirs import CIRS
 from .ecliptic import *  # there are a lot of these so we don't list them all explicitly
```

Comment:
```suggestion
```

Reply:
Resolved for now by https://github.com/astropy/astropy/pull/13853#discussion_r1002588599.
In the future, this should be re-examined. There's #13785 for further discussion.